### DISCUSSION TITLE
增加keyword提示

### DISCUSSION THREAD

**Discussion by Extra-Creativity** [xmake-io/xmake-vscode#254] [Q&A] [answered]
感觉VSCode插件对新版本的XMake好多函数都没提示，比如`set_encodings`，可以一次性全加进去。不知道其他IDE或者编辑器的插件有没有这个问题。

**Comment by waruqi** [comment, marked as answer]
可以提个 pr 过来更新下。https://github.com/xmake-io/xmake-vscode/blob/master/src/commands.ts
